sections/semantics-embedded-content.include

```diff
@@ -6124,8 +6124,8 @@ zero or more <{track}> elements, then
   var support = videoElement.canPlayType('video/x-new-fictional-format;codecs="kittens,bunnies"');
   if (support != "probably") {
     // not confident of browser support
-    // but we have a plugin
-    // so use plugin instead
+    // but we might have a plugin
+    // so try plugin instead
```

Comment:
Yeah, see above comment. I think I'll reserve plugin use to the empty string case. The browser, not supporting any plugins (that it knows about--perhaps it stops caring about "installed" plugins in the same way it does now) will only be answering canPlayType for the things it knows about... so, an empty string response means still try a plugin cause you never know...

Maybe this logic is false though... open to alternatives (including removing the example altogether...)